index.html

```diff
@@ -8,181 +8,188 @@
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <h3 class="social">LET’S CONNECT</h3>
+                <ul class="socialIcon">
+                    <li><a href="#"><img class="icon" src="img\Vector.svg" alt="Tweeter icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\Vector m.svg" alt="M icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\github icon.svg" alt="github icon"></a></li>
+                    <li><a href="#"><img class="icon" src="img\peace icon.svg" alt="peace icon"></a></li>
                 </ul>
             </section>
-            <p>
-                A daily selection of privately personalized 
-                reads; no accounts or sign-ups required.
-            </p>
-            <ul class="programmingLanguages">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
-            <button class="button">See Project</button>
         </section>
-        <!-- projecto 2 -->
-        <section class="project">
-            <img class="snap" src="img\Snapshoot Portfolio 2.svg" alt="Project picture">
-            <h2>Multi-Post Stories</h1>
-            <section>
-                <ul class="canopy">
-                    <li>CANOPY</li>
-                    <li><img src="img\Counter.svg"></li>
-                    <li class="item">Back End Dev</li>
-                    <li><img src="img\Counter.svg"></li>
-                    <li class="item">2015</li>
+        <section id="work">
+            <!-- projecto 1 -->
+            <section class="project">
+                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="Project picture">
+                <h2>Tonic</h1>
+                <section>
+                    <ul class="canopy">
+                        <li>CANOPY</li>
+                        <li><img src="img\Counter.svg"></li>
+                        <li><span class="lightgray">Back End Dev</span></li>
+                        <li><img src="img\Counter.svg"></li>
+                        <li><span class="lightgray">2015</span></li>
+                    </ul>
+                </section>
+                <p>
+                    A daily selection of privately personalized 
+                    reads; no accounts or sign-ups required.
+                </p>
+                <ul class="programmingLanguages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
                 </ul>
+                <button class="button">See Project</button>
             </section>
```

Comment:
- [ ] Kindly decrease the height of the project cards to match the design on Figma. It will also make it look better for smaller screens because right now the cards look way too big in smaller screens 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199976944-cb7e39c3-30c0-469a-ba0b-130b5669927b.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199976973-ceb6ceec-0b7b-4743-a5a1-96454fa076e8.png)